src/qt/bitcoin.cpp

```diff
@@ -480,6 +481,11 @@ int GuiMain(int argc, char* argv[])
         return EXIT_SUCCESS;
     }
 
+    if (gArgs.IsArgSet("-qtinfo")) {
+        tfm::format(std::cout, "%s", GUIUtil::QtInfo().toStdString());
```

Comment:
I'm not 100% familiar with `GUIItl::QtInfo()` but maybe it is better to print a new line after `%s`

Reply:
A new line is redundant here. Please see:
https://github.com/bitcoin/bitcoin/blob/5ea7c911fc7630038fc3429018270a6a38facf45/src/qt/guiutil.cpp#L900

Also the same approach is used here:
https://github.com/bitcoin/bitcoin/blob/392edbd49abec4a8c587979dbb1f44e8605b7ce1/src/bitcoind.cpp#L74